functions/src/index.test.ts

```diff
@@ -0,0 +1,5 @@
+describe("Hello World", () => {
+    test("Placeholder Test", () => {
```

Comment:
Out of curiosity, do we have prettier configured with 4-space indents?  Or is this file not getting formatted or something?

Reply:
Ah, we haven't set up a prettier pre-commit hook, so the file isn't getting formatted. I just ran it manually in 8e3e63e695a84ed5cc8bda407dae759632c3a09e but I'll add a ticket to add that